aws_lambda_powertools/utilities/data_classes/api_gateway_proxy_event.py

```diff
@@ -319,14 +323,12 @@ def header_serializer(self):
         return HttpApiHeadersSerializer()
 
     @property
-    def resolved_query_string_parameters(self) -> Optional[Dict[str, Any]]:
+    def resolved_query_string_parameters(self) -> Optional[Dict[str, List[str]]]:
         if self.query_string_parameters is not None:
-            query_string = {
-                key: value.split(",") if "," in value else value for key, value in self.query_string_parameters.items()
-            }
+            query_string = {key: value.split(",") for key, value in self.query_string_parameters.items()}
             return query_string
 
-        return {}
+        return None
```

Comment:
perhaps `{}` so you remove additional checks on `None` and type annotations `Optional` upstream?